rag/advanced_rag/knowlege_compile/wiki_incremental.py

```
@@ -0,0 +1,1619 @@
+    existing_map = settings.docStoreConn.get_fields(existing, ["id", "page_ids", "source_chunk_hashes"])
+
+    doc = {
+        "doc_id": doc_id,
+        "kb_id": kb_id,
+        "page_ids": json.dumps(page_ids, ensure_ascii=False),
+        "source_chunk_hashes": json.dumps(chunk_hashes or {}, ensure_ascii=False),
+        "map_checksum": map_checksum or "",
+        "compile_kwd": WIKI_DOC_PAGE_SOURCE_COMPILE_KWD,
+    }
+    if existing_map:
+        for row in existing_map.values():
```

Ruff (0.16.0)

[warning] Loop control variable `row` not used within loop body

Rename unused `row` to `_row`

(B007)